### PR TITLE
CMake MSVC version resource support

### PR DESCRIPTION
**Pull request by remoe**
solves #718

### CODE REVIEW

File: lib/version.rc.in [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	#define        VER_STR  "@PROJECT_VERSION@.0 (MSVC release)"
+	#define        DBG      ""
+	FILEFLAGS      0x0L
+#endif
+BEGIN
+BLOCK "StringFileInfo"
+BEGIN
+  BLOCK "040904b0"
+  BEGIN
+	VALUE "CompanyName",      "http://tatsuhiro-t.github.io/nghttp2/"
```
Comment (tatsuhiro-t) [inline lib/version.rc.in]: Could you use https://nghttp2.org/ here?

**Comment by tatsuhiro-t** [comment]
Thank you for PR.  I found one minor thing.  Could you fix that up?

**Comment by remoe** [comment]
ok, done.

**Comment by tatsuhiro-t** [comment]
Thank you.  Cherry picked as ee84404

**Pull request closed without merging**